accounting/README.md

```diff
@@ -12,10 +12,18 @@ transport connection.
 Accounting Records are available at a gNMI origin:
    gnmi.accounting
 
+Records maybe streamed from a system either at request of the remote
+caller, via the ClientStream() rpc, or at the request of the system
+to a remote endpoint, via the ServerStream() rpc.
```

Comment:
update?

Reply:
Ideally the structure is correct now, I'm sorting out documentation currently.